### DISCUSSION TITLE
Testing Claude 3 API: reply is in a different language then the question

### DISCUSSION THREAD

**Discussion by FamedBear16** [logancyang/obsidian-copilot#356] [General]
I am based in France, but when I ask Claude 3 a question in English he replies in French.
Why is the API answering me in French? (GTP-4 Turbo replies in English)

I have also noticed that the QA modes break when I insert the Claude 3 API in settins.. (I have reported this behaviour in bugs).

Regards,

**Comment by xinyu-dev** [comment]
Same issue here with Claude 3 API. Except for English I don't have any language in my notes. It responds in different languages

**Comment by logancyang** [comment]
GPT-3.5 also does it at times. I personally haven't seen Claude 3 does it but it's entirely possible. Obsidian Copilot does nothing but sending the note to the API with a one-liner default system prompt "I'm obisidan copilot ...". The behavior is entirely up to the model.

I recommend a custom system prompt in Copilot setting with "reply in English" or something like that.
